app/assets/stylesheets/posts.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Posts controller here.
```

Comment:
delete this file because not use

Reply:
![Screenshot from 2021-08-25 13-55-01](https://user-images.githubusercontent.com/87750131/130734238-f9d08375-f694-41a8-b78d-8eb3ba4518bc.png)
I hope this is correct. attached u with photo